### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/image/commit/88bd34a9d64afb4ad9a36cd0a924a484367667d0#commitcomment-45406236

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/image/commit/e5a46759ad6e7f08cb3b8cb40d325ea6718246b0

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: docker/reference/reference_test.go
Comment (jsoref) [inline docker/reference/reference_test.go:388]: I don't have a preferred spelling in this family, but it felt like this repo used this spelling more for this specific flavor (`-ing`).

File: manifest/fixtures/v2s1-invalid-signatures.manifest.json
Comment (jsoref) [inline manifest/fixtures/v2s1-invalid-signatures.manifest.json:3]: I could be wrong

File: signature/policy_eval_signedby_test.go
Comment (jsoref) [inline signature/policy_eval_signedby_test.go:126]: Not sure about the `..` ...

(I can change it if requested)

**Review by jsoref — COMMENTED** [review]
I leave my changes as distinct commits initially as it makes it easier to address review comments (typically dropping a word family, or when I guess wrong picking a different correction for the family).

I'm happy to squash once people are happy w/ the underlying changes.

File: manifest/fixtures/v2s1-invalid-signatures.manifest.json
Comment (mtrmac) [reply manifest/fixtures/v2s1-invalid-signatures.manifest.json:3]: You’re right.

File: signature/policy_eval_signedby_test.go
Comment (mtrmac) [reply signature/policy_eval_signedby_test.go:126]: Looking at the other `, kind of,` comments around, that was intended to be just a sentence-ending period.

File: manifest/fixtures/v2s1-invalid-signatures.manifest.json
Comment (mtrmac) [reply manifest/fixtures/v2s1-invalid-signatures.manifest.json:3]: Note to self: These kind of changes do affect some of the digest constants, and I have _not_ reviewed that so far.

File: internal/testing/explicitfilepath-tmpdir/tmpdir.go
Comment (mtrmac) [inline internal/testing/explicitfilepath-tmpdir/tmpdir.go:11]:
```suggestion
// import _ "github.com/containers/image/internal/testing/explicitfilepath-tmpdir"
```

File: pkg/sysregistriesv2/system_registries_v2.go
Comment (mtrmac) [inline pkg/sysregistriesv2/system_registries_v2.go:33]: Note to self:
```suggestion
// -ldflags '-X github.com/containers/image/sysregistries.systemRegistriesConfDirPath=$your_path'
```
, or rather `containers/image/v5/…` IIRC, actually. That’s out of scope though.

**Comment by rhatdan** [comment]
@jsoref Could you rebase so we can get this merged?

File: pkg/sysregistriesv2/system_registries_v2.go
Comment (jsoref) [reply pkg/sysregistriesv2/system_registries_v2.go:33]: I think I'm going to leave this to you, it's too complicated to count as spelling.

**Comment by rhatdan** [comment]
Tests are failing?

**Comment by rhatdan** [comment]
Tests are failing?
I am not a big fan of this many commits, I would prefer them squashed, but I will leave that up to @mtrmac

**Comment by jsoref** [comment]
Someone needs to give the tests a docker token so it doesn't hit a pull rate limit:
> FAIL: copy_test.go:784: CopySuite.TestCopySignatures
Running skopeo --policy /tmp/policy.json929996936 copy docker://busybox:latest dir:/tmp/signatures-dest263441037
Running skopeo --policy /tmp/policy.json929996936 copy docker://openshift/hello-openshift dir:/tmp/signatures-dest263441037
Running skopeo --tls-verify=false copy --sign-by personal@example.com docker://busybox:1.26 atomic:localhost:5006/myns/personal:personal
copy_test.go:809:
    // type: signedBy
    // Sign the images
    assertSkopeoSucceeds(c, "", "--tls-verify=false", "copy", "--sign-by", "personal@example.com", "docker://busybox:1.26", "atomic:localhost:5006/myns/personal:personal")
utils.go:62:
    c.Assert(err, check.IsNil, check.Commentf("%s", out))
... value *exec.ExitError = &exec.ExitError{ProcessState:(*os.ProcessState)(0xc0004d25c0), Stderr:[]uint8(nil)} ("exit status 1")
... time="2021-01-20T15:19:53Z" level=warning msg="'--tls-verify' is deprecated, please set this on the specific subcommand"
time="2021-01-20T15:20:24Z" level=fatal msg="Error initializing source docker://busybox:1.26: Error reading manifest 1.26 in docker.io/library/busybox: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit"

Fixing that is well beyond the scope of this PR.

I'm totally fine w/ squashing once people are ok w/ a PR. I'm also quite happy to let the project do the squashing. I keep things unsquashed while developing because it makes rebasing *much* easier.

**Comment by jsoref** [comment]
For reference, that problem is reported as #1107

**Comment by rhatdan** [comment]
@jsoref Could you rebase and repush this PR?

**Comment by jsoref** [comment]
@rhatdan: done

**Comment by rhatdan** [comment]
LGTM
@vrothberg @mtrmac @lsm5 PTAL

**Review by lsm5 — APPROVED** [review]
woah woah .. LGTM! :)

**Comment by jsoref** [comment]
If you like the results, the tool I use to develop these PRs can be used to prevent typos from being added to the repository later :-).

**Comment by lsm5** [comment]
> If you like the results, the tool I use to develop these PRs can be used to prevent typos from being added to the repository later :-).

might be good to add to the CI itself.

**Comment by lsm5** [comment]
@cevich do you know whatsup with $GCPJSON at https://cirrus-ci.com/task/5693825629814784?command=main#L1 ?

**Comment by jsoref** [comment]
@lsm5 it's a github action, it looks something like this: https://github.com/jsoref/image/blob/88bd34a9d64afb4ad9a36cd0a924a484367667d0/.github/workflows/spelling.yml (plus the other files in that commit which tune it)

**Review by vrothberg — COMMENTED** [review]
LGTM, thank you

**Comment by cevich** [comment]
> @cevich do you know whatsup with $GCPJSON at https://cirrus-ci.com/task/5693825629814784?command=main#L1 ?

Ohh! That's bad.  Investigating.

**Comment by cevich** [comment]
Well hmmm...that critical task [ran fine on master, after merging this.](https://github.com/containers/image/runs/2183996802).  Best I have at the moment is it was some kind of value-decryption anomaly on the Cirrus-CI side.  I'll keep an eye out for this happening more/elsewhere.  Thanks very much @lsm5 for bringing it to my attention.

**Pull request merged**
